editor/src/components/canvas/canvas-strategies/canvas-strategy-types.ts

```diff
@@ -12,24 +12,24 @@ export interface CustomStrategyState {
   escapeHatchActivated: boolean
   lastReorderIdx: number | null
   duplicatedElementNewUids: { [elementPath: string]: string }
+  reparentedToPaths: Array<ElementPath>
 }
 
-export function defaultCustomStrategyState(): CustomStrategyState {
-  return {
-    escapeHatchActivated: false,
-    lastReorderIdx: null,
-    duplicatedElementNewUids: {},
-  }
+export const defaultCustomStrategyState: CustomStrategyState = {
```

Comment:
Can you `deepFreeze` this please so that we catch any accidental mutations that might happen